pontoon/base/models.py

```diff
@@ -1150,6 +1150,16 @@ class Project(AggregatedStats):
         Allow localizers to request the project for their team.
     """)
 
+    configuration_file = models.CharField(
+        null=True,
+        blank=True,
+        max_length=2000,
+        help_text="""
+        A path to the optional project configuration file, relative to the
+        source string repository.
```

Comment:
nit: this should be indented one more block.

Reply:
Huh, we keep making this mistake consistenly across the entire `models.py` file. Maybe we should take care of it as part of https://bugzilla.mozilla.org/show_bug.cgi?id=1415732.